locales/en-US/messages.properties

```diff
@@ -155,6 +156,7 @@ navDarkThemeIconTitle=Switch to the dark theme
 navLightThemeIconTitle=Switch to the light theme
 navWrapTextTitle=Toggle wrapping long lines of code
 navAllowJSTitle=Toggle running JavaScript code in the preview
+navXMLSVGTitle=Toggle openning svg file with code editor or image viewer
```

Comment:
Given that this toggle is an on/off, we can say what this does when turned on: `View SVG files as code instead of as image`